FreeRTOS_IP.c

```diff
@@ -41,6 +41,8 @@
 
 /* FreeRTOS+TCP includes. */
 #include "FreeRTOS_IP.h"
+#include "FreeRTOS_ICMP.h"
+#include "FreeRTOS_IP_Timers.h"
 #include "FreeRTOS_Sockets.h"
```

Comment:
Missing: `#include "FreeRTOS_IP_Utils.h"` 
If not, `prvProcessNetworkDownEvent()` and other identifiers would be "undeclared".